src/main/java/org/cactoos/text/TextOf.java

```diff
@@ -399,6 +403,210 @@ public TextOf(final LocalDate date, final DateTimeFormatter formatter) {
+     * @param date The date to format.
+     */
+    public TextOf(final LocalDateTime date) {
+        this(date, new Iso().value());
+    }
+
+    /**
+     * Formats date using provided date time format string using default locale.
+     * @param date The date to format.
+     * @param format The format to use.
+     */
+    public TextOf(final LocalDateTime date, final String format) {
+        this(date, format, Locale.getDefault(Locale.Category.FORMAT));
+    }
+
+    /**
+     * Formats the date using the provided format string using the provided
+     * locale.
+     * @param date The date to format.
+     * @param format The format string to use.
+     * @param locale The locale to use.
+     */
+    public TextOf(final LocalDateTime date, final String format,
+        final Locale locale) {
+        this(date, DateTimeFormatter.ofPattern(format, locale));
+    }
+
+    /**
+     * Formats the milliseconds using the ISO format.
+     * @param milliseconds Milliseconds to format.
+     */
+    public TextOf(final Long milliseconds) {
+        this(
+            Date.from(Instant.ofEpochMilli(milliseconds)),
+            new Iso().value()
+        );
+    }
+
+    /**
+     * Formats the milliseconds using the format and the default locale.
+     * @param milliseconds Milliseconds to format.
+     * @param format The format to use.
+     */
+    public TextOf(final long milliseconds, final String format) {
+        this(
+            Date.from(Instant.ofEpochMilli(milliseconds)), format,
+            Locale.getDefault(Locale.Category.FORMAT)
+        );
+    }
+
+    /**
+     * Formats the milliseconds as date using the format and the locale.
+     * @param milliseconds Milliseconds to format as date.
+     * @param format The format to use.
+     * @param locale The locale to use for the format.
+     */
+    public TextOf(final long milliseconds, final String format,
+        final Locale locale) {
+        this(
+            Date.from(Instant.ofEpochMilli(milliseconds)),
+            DateTimeFormatter.ofPattern(format, locale)
+        );
+    }
+
+    /**
+     * Formats the date with ISO format using the system zone.
+     * @param date The date to format.
+     */
+    public TextOf(final Date date) {
+        this(date, new Iso().value());
+    }
+
+    /**
+     * Formats the date with to format using the default locale and the system
+     * zone.
+     * @param date The date to format.
+     * @param format The format to use.
+     */
+    public TextOf(final Date date, final String format) {
+        this(date, format, Locale.getDefault(Locale.Category.FORMAT));
+    }
+
+    /**
+     * Formats the date using the format and locale using the system default
+     * zone.
+     * @param date The date to format.
+     * @param format The format to use.
+     * @param locale The locale to use.
+     */
+    public TextOf(final Date date, final String format,
+        final Locale locale) {
```

Comment:
Line wrapping is inconsistent with other classes, e.g. `FormattedText`. I think we should use the ["bracket should either start/end a line or be paired on the same line"](https://www.yegor256.com/2014/10/23/paired-brackets-notation.html) rule (as in `FormattedText`).

So in this case it would look like this:

```java
public TextOf(
    final Date date,
    final String format,
    final Locale locale
) {
    . . .
}
```

This comment applies to other constructors as well.